### PR TITLE
Fixes on DAU and MAU aggregations

### PR DESCRIPTION
**Pull request by rodrigok**
- [x] Aggregate OS for last month
- [x] Aggregate Devices for last month
- [x] Count sessions correctly for devices
- [x] Count unique sessions correctly
- [x] Add unit tests
- [x] Report new data
- [x] Change for past 30 days
- [x] Use monogdb container for Unit Tests on CI (isn't possible to use mongoUnit for MongoDB 3.2 in Debian 9.2)
- [x] Report Electron client correctly
- [x] Check compatibility with stats collector (https://github.com/RocketChat/Rocket.Chat.StatisticsCollector/pull/36)
- [x] Migration
- [x] Test migration with real data

### CODE REVIEW

**Comment by mrsimpson** [comment]
Good to see this! I consider activity measurements as essential for a successful SaaS.
But I don't think the information "Server has seen the user" sufficient to declare the user "active". This information can also be quite easily taken from a Piwik / Google Analytics.
What I consider interesting is how many rooms have been read and how many messages have been written in the interval. We implemented it quite simply here: https://github.com/assistify/Rocket.Chat/blob/develop/packages/rocketchat-statistics/server/functions/getUsages.js
Would love to read you opinion!

**Comment by rodrigok** [comment]
Thanks @mrsimpson it looks interesting, will check tomorrow 😄

File: app/models/server/models/Sessions.js [hunk 1]
```diff
@@ -1,5 +1,374 @@
 import { Base } from './_Base';
 
+export const aggregates = {
```
Comment (sampaiodiego) [inline app/models/server/models/Sessions.js]: any reason to export a whole object instead of individual functions?

Comment (rodrigok) [reply app/models/server/models/Sessions.js]: Just to be more organized and clear about what the methods does

Comment (sampaiodiego) [reply app/models/server/models/Sessions.js]: IMO it is more organized when you export individual functions.. function names should be clear enough for explanation =) 

or should they be static methods of the `Sessions` class? idk.

**Comment by rodrigok** [comment]
@mrsimpson your code is interesting, but I'm not sure what `readsAggregationCursor` should means, are you getting the new subscriptions after last statistics?

**Pull request merged**